libpod/runtime_pod_pause_linux.go

```diff
@@ -0,0 +1,234 @@
+// +build linux
+
+package libpod
+
+import (
+	"context"
+	"fmt"
+	"os"
+	"path/filepath"
+	"time"
+
+	"github.com/containers/image/storage"
+	"github.com/containers/image/transports/alltransports"
+	"github.com/containers/image/types"
+	"github.com/containers/storage/pkg/archive"
+	digest "github.com/opencontainers/go-digest"
+	"github.com/opencontainers/runtime-tools/generate"
+	"github.com/pkg/errors"
+	"github.com/projectatomic/buildah"
+	"github.com/projectatomic/buildah/util"
+	"github.com/projectatomic/libpod/libpod/image"
+	"github.com/sirupsen/logrus"
+)
+
+var (
+	pauseCmd      = "/pause"
+	pauseLocation = "bin/pause"
+	pauseImgName  = "pause-infra-container"
+)
+
+func (r *Runtime) buildPauseImage(ctx context.Context, p *Pod) (string, error) {
+	// Buildah new
+	// TODO make sure this matches global cli.context
+	signaturePolicy := ""
+	store := r.store
+	commonOpts := &buildah.CommonBuildOptions{}
+	transport := DefaultTransport
+	isolation := buildah.IsolationDefault
+	sc := &types.SystemContext{}
+	pullPolicy := buildah.PullNever
+	namespaceOptions := make(buildah.NamespaceOptions, 0, 7)
+
+	builderOptions := buildah.BuilderOptions{
```

Comment:
Wait, why are we calling into buildah for this?

We just need an OCI spec, we shouldn't need buildah for that.